src/app/fdctl/run/tiles/fd_quic.c

```diff
@@ -127,16 +119,15 @@ legacy_stream_notify( fd_quic_ctx_t * ctx,
   fd_tpu_reasm_slot_t * slot   = fd_tpu_reasm_prepare( ctx->reasm, tsorig );
 
   int add_err = fd_tpu_reasm_append( ctx->reasm, slot, packet, packet_sz, 0UL );
-  ctx->metrics.legacy_reasm_append[ add_err ]++;
-  if( FD_UNLIKELY( add_err!=FD_TPU_REASM_SUCCESS ) ) return;
+  if( FD_UNLIKELY( add_err!=FD_TPU_REASM_SUCCESS ) ) return; /* unreachable */
 
   uint   tspub = (uint)fd_frag_meta_ts_comp( fd_tickcount() );
   void * base  = ctx->verify_out_mem;
   ulong  seq   = stem->seqs[0];
 
   int pub_err = fd_tpu_reasm_publish( ctx->reasm, slot, stem->mcaches[0], base, seq, tspub );
-  ctx->metrics.legacy_reasm_publish[ pub_err ]++;
-  if( FD_UNLIKELY( pub_err!=FD_TPU_REASM_SUCCESS ) ) return;
+  if( FD_UNLIKELY( pub_err!=FD_TPU_REASM_SUCCESS ) ) return; /* unreachable */
+  FD_MCNT_INC( QUIC_TILE, TXNS_RECEIVED_UDP, 1UL );
```

Comment:
This success case is a lot of cache traffic. We typically update the actual metric in housekeeping for these frequent ones.